pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,26 @@
+{ pkgs }:
+
+pkgs.stdenv.mkDerivation {
```

`pkgs` in the arguments makes this a lot harder to override, and I believe it might have other downsides as well. Let's expand this argument list.

Also, there doesn't seem to be any need for a C compiler toolchain, so we can use `stdenvNoCC`.

See my comment regarding `fetchzip`

```suggestion
{ stdenvNoCC
, fetchzip
, gtk3
}:

stdenvNoCC.mkDerivation {
```